### DISCUSSION TITLE
AWS installation removed from marketplace?

### DISCUSSION THREAD

**Discussion by EnginEren** [mozilla/hubs#4209] [Hubs Cloud Questions] [answered]
![Screenshot from 2021-04-30 10-05-34](https://user-images.githubusercontent.com/11681207/116667502-8e295d80-a99c-11eb-955b-79f63847dcf4.png)
Dear all,

I was surprised to see that I cannot install Hubs Cloud personal since that page says that this version was removed [*]

Cheers,
Engin

[*] https://aws.amazon.com/marketplace/pp/B084RZH56R

**Comment by brianpeiris** [comment]
Hi Engin. 
We are aware of the situation and are looking into resolving it. We will update as soon as we can. 
Thank you for the report.

**Comment by brianpeiris** [comment, marked as answer]
Hi Engin. Hubs Cloud is now available again on the AWS marketplace. Let us know if you run into any issues.

**Reply by EnginEren** [reply]
Hi Brian, thanks a lot for the information. Great to see that it's available again!
Cheers,
